protocol/devops/README.md

```diff
@@ -0,0 +1,75 @@
+# DevOps Protocol
+
+A set of guidelines, checklists and tutorials for deploying web apps in a sane
+and reproduceable way.
+
+## Table of Contents
+
+* [Continuous Integration](#continuous-integration)
+* [Servers](#server-setup)
+* [Static websites](#static-websites)
+* [DNS](#server-setup)
+* [SSL](#server-setup)
+    * [Sabayon Samples](#sabayon-samples)
+
+## Continuous Integration
+
+* Use [SemaphoreCI] ([TravisCI] is also fine for open source, but tends to be
+    slower)
+* Set up automatic deploys to staging (from the `master` branch), and manual
+    deploys to production
+* Ensure the CI server is capable of deploying to all environments, so that no
```

Comment:
The CLI is exactly the point I want to avoid.
I want the process to involve the least configuration possible for anyone in the team. Deploying via CLI implies that you have the CLI, know how to use it, and have access to the heroku app.

None of these are true for non-developers or developers that don't belong to a particular project

Note that I'm not against using it, quite the opposite. I just want to enforce that, for each project, there's a preconfigured way that anyone can use to deploy